streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -841,12 +838,37 @@ public void apply(final StreamTask task) {
         if (e != null) {
             throw e;
         }
+    }
 
+    private void commitAllStandbyTasks() {
+        RuntimeException e = null;
         for (final StandbyTask task : standbyTasks.values()) {
-            commitOne(task);
+            try {
+                commitOne(task);
+            } catch (final RuntimeException t) {
```

Comment:
What kind of `RuntimeException` do we expect here? Should we maybe catch those within `commitOne` ?

Also, we have a different pattern with an loop instead of `performOnStreamTask` as in `commitAllActiveTasks` -- should we use the same pattern for both?